docs/gtfs-bus-integration-research.md

```
@@ -0,0 +1,545 @@
+
+| 項目 | 技術 |
+|------|------|
+| 言語 | Rust (edition 2021) |
+| 非同期ランタイム | Tokio |
+| API | gRPC (Tonic) + gRPC-Web |
+| データベース | PostgreSQL 18 |
+| ORM | SQLx |
+
+### 1.2 データモデル
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)